From 87d8aec02a5b9b976d0591c6fe0cf31da7ffdfa5 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Fri, 8 May 2020 18:22:24 +0300
Subject: [PATCH 47/51] cl-som-imx8: Update config.h

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 include/configs/cl-som-imx8.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/include/configs/cl-som-imx8.h b/include/configs/cl-som-imx8.h
index 7ea0e65216..32aea8918b 100644
--- a/include/configs/cl-som-imx8.h
+++ b/include/configs/cl-som-imx8.h
@@ -32,7 +32,6 @@
 #define CONFIG_SPL_STACK		0x187FF0
 #define CONFIG_SPL_LIBCOMMON_SUPPORT
 #define CONFIG_SPL_LIBGENERIC_SUPPORT
-#define CONFIG_SPL_SERIAL_SUPPORT
 #define CONFIG_SPL_GPIO_SUPPORT
 #define CONFIG_SPL_MMC_SUPPORT
 #define CONFIG_SPL_BSS_START_ADDR      0x00180000
@@ -163,8 +162,6 @@
 
 #define PHYS_SDRAM_SIZE			0x40000000
 
-#define CONFIG_NR_DRAM_BANKS		2
-
 #define CONFIG_SYS_MEMTEST_START    PHYS_SDRAM
 #define CONFIG_SYS_MEMTEST_END      (CONFIG_SYS_MEMTEST_START + (PHYS_SDRAM_SIZE >> 1))
 
-- 
2.11.0

